### DISCUSSION TITLE
How is this project different and more capable than ranger

### DISCUSSION THREAD

**Discussion by Greatz08** [sxyazi/yazi#806] [1. Q&A]
please if possible explain in detail about all unique points

**Comment by lpnh** [comment]
Hi! Have you had a chance to check out Yazi's documentation page? The [Features](https://yazi-rs.github.io/features/) and [Tips](https://yazi-rs.github.io/docs/tips) sections offer a good overview of Yazi's capabilities. The pinned [Feature requests](https://github.com/sxyazi/yazi/issues/51) issue can be a good place to look as well. Finally, you can give it a try. Then, it's up to you to judge whether it's less or more capable than other available tools. If you're looking for more in-depth design details, the [Blog](https://yazi-rs.github.io/blog) section will certainly provide insightful information.

**Comment by bvr-yr** [comment]
ranger is ramshackle relic of the past, constantly blocking whole UI

I think this is a main advantage

Feature parity will be reached soon (if not already) as yazi is developing super fast
Though ranger is more user friendly (I guess?) cause of python + bash, which you can hack and extend much easier than rust + lua

**Comment by GianniBYoung** [comment]
This is a ton faster and image previews seem to be alot easier to get working. Also lua for plugins seems to be taking of thanks to neovim
